python/tests/tsdf_tests.py

```diff
@@ -619,17 +619,6 @@ def test_group_stats(self):
         self.assertDataFramesEqual(featured_df, dfExpected)
 
 
-class UtilsTest(SparkTest):
```

Comment:
Why are we removing this class? It doesn't seem related the functionality changes in this PR.